assembl/scripts/ini_combine.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+from argparse import ArgumentParser, FileType
+from ConfigParser import SafeConfigParser as Parser
+from os import urandom
+from base64 import b64encode
+
+
+if __name__ == '__main__':
+    ap = ArgumentParser(description='Combine two ini files.')
+    ap.add_argument('source', type=FileType('r'),
+                    help='The source file')
+    ap.add_argument('overlay', type=FileType('r'),
+                    help='The overlay file')
+    ap.add_argument('result', type=FileType('w'),
+                    help='The resulting file')
+    args = ap.parse_args()
+    source = Parser()
```

Comment:
Lots of second thought about this one. Still think it's a good basic idea, but I'm not sure I see myself using it quite yet.
Using the ArgumentParser means that it's _not_ able to be used to modify a target in-place, btw; but we can use '-' as argument for source, overlay and/or result, meaning stdin/stdout. (not source and overlay both, of course.) (Adding that to a comment...)
But now, I'm thinking more in terms of a templatized production.ini, it's a different direction.